forc-plugins/forc-doc/src/render.rs

```diff
@@ -350,8 +446,16 @@ fn qualified_file_path(module_prefix: &Vec<String>, file_name: String) -> String
 
     file_path.to_str().unwrap().to_string()
 }
-
-fn docs_to_html(attributes: &AttributesMap) -> String {
+/// Create a path prefix string for navigation from the `module_depth`
+fn module_depth_to_path_prefix(module_depth: usize) -> String {
+    let mut prefix = String::new();
+    for _ in 0..module_depth {
+        prefix.push_str("../")
+    }
+    prefix
```

Comment:
Maybe we could re-write like the following to shorten a little: 

```suggestion
    (0..module_depth).map(|_| "../").collect::<String>()
```

Reply:
I like this a lot more, just didn't want to write a while loop haha